### DISCUSSION TITLE
Installing AudioGridder to Mac OSX 10.7.5 for dummies... (me)

### DISCUSSION THREAD

**Discussion by TexTiles2** [apohl79/audiogridder#1320] [Q&A]
Hi there, when I attempt to open AG server 1.1.1 on my older macpro AG says "Warning AudioGridder needs the Accessibility permission to remote control plugins." and then goes into a loop and have to force quit.

I don't think there is an "Accessibility permission" on 10.7.5. to my understanding. When I google it I've seen it stated that that was introduced in OSX 10.14

I thought 10.7 was supported. A step by step instruction for installation and setup with the network would be great for dummies like me. 

Any guidance would be greatly appreciated. I have searched extensively but not found any solutions that way.

Thanks very much.

**Comment by emack999** [comment]
Says on the AG download page 10.8+
![ScreenHunter 26](https://github.com/apohl79/audiogridder/assets/81657685/20f08528-f4f9-41c5-a2a6-b3e6fc8b13c8)

**Reply by TexTiles2** [reply]
Thanks emac999, if you scroll down the installer download page you see the 10.7 version. I downloaded the installer for 10.7-x86_64. Thanks for the reply. I wonder if he mistakenly changed the download for 10.7 to 10.8 when updating.

My macpro is full of mixed UAD1 and 2 cards and the last supported OS is 10.7 for UAD1 software, so that is my conundrum.

I had thought of putting the UAD1 cards in one macpro and the UAD2 card in the second (I have two) with a newer OS and if this worked I'd have 16 cores, or possibly 20 if i ran it off my laptop.  (plus the UAD cores)

It would be a gamechanger if I could get this to work, as many of the older plugins I have and use daily are obsolete on newer systems.
![audiogridder downloads](https://github.com/apohl79/audiogridder/assets/157448791/b775ac3c-725e-4b84-a69d-fb8d570af54a)

**Comment by TexTiles2** [comment]
So, although it says on the website it is for mac os 10.7, there doesn't seem to be any way to enable "Accessibility" in 10.7.

So I updated this drive that I prepared for this experiment to 10.8. It is the same, no way that I can see to enable Accessibility so no worky.

I have another drive with 10.11 and it does have a spot where you can allow audiogridder to control the computer but still no connection, although it did load on 10.11 but still no worky.

Is there anyone actually getting this working on 10.7 or 10.8?
